packages/scripts/package.json

```diff
@@ -1,10 +1,15 @@
 {
   "name": "@masknet/cli",
   "private": true,
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "bin": {
     "build": "./src/bin/build.ts",
     "dev": "./src/bin/dev.ts"
   },
+  "scripts": {
+    "postinstall": "tsc"
```

Comment:
Need to compile package after install (is like @masknet-shared compiled when `dev`), otherwise IDE will complaint`Cannot find module '@masknet/cli' or its corresponding type declarations.ts(2307)`